src/test/java/net/snowflake/client/jdbc/ResultSetLatestIT.java

```diff
@@ -971,4 +979,70 @@ public void testLargeStringRetrieval() throws SQLException {
       fail("executeQuery should not fail");
     }
   }
+
```

Comment:
I wasn't sure if for the tests with JSON result format I needed to include the new jdbc_arrow_treat_decimal_as_int property since it shouldn't have any effect but I've included it for now to mirror the test cases for ARROW result format. Please let me know if we don't care about testing the new connection property for JSON and I can change the test.

Reply:
arrow related property does not need to be included in json tests - it does not influence the json result set